src/inject/dynamic-theme/css-rules.ts

```diff
@@ -7,7 +7,14 @@ export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule
     forEach(rules, (rule) => {
         if (rule instanceof CSSMediaRule) {
             const media = Array.from(rule.media);
-            if (media.includes('screen') || media.includes('all') || !(media.includes('print') || media.includes('speech'))) {
+            media.map((mediaRule) => {
+                if (mediaRule.includes(' and ')) {
+                    mediaRule.split(' and ').forEach((splittedRule) => media.push(splittedRule));
+                } else {
+                    return mediaRule;
+                }
+            });
+            if (media.includes('screen') || media.includes('all') || !((media.includes('print') && (media.length !== 1 && !matchMedia(rule.media.mediaText).matches)) || media.includes('speech'))) {
```

Comment:
I would suggest to make this condition more readable and utilise `some()` method. The last condition is hard to understand, I'm not sure matchMedia should be there:
```typescript
const isScreenOrAll = media.some((m) => m.startsWith('screen') || m.startsWith('all'));
const isPrintOrSpeech = ...
if (isScreenOrAll || !isPrintOrSpeech) ...
```